rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -358,7 +358,7 @@ http {
     {{ range $name, $upstream := $backends }}
     {{ if eq $upstream.SessionAffinity.AffinityType "cookie" }}
     upstream sticky-{{ $upstream.Name }} {
-        sticky hash={{ $upstream.SessionAffinity.CookieSessionAffinity.Hash }} name={{ $upstream.SessionAffinity.CookieSessionAffinity.Name }}  httponly;
+        sticky hash={{ $upstream.SessionAffinity.CookieSessionAffinity.Hash }} name={{ $upstream.SessionAffinity.CookieSessionAffinity.Name }}{{ range $locationName, $locationPaths := $upstream.SessionAffinity.CookieSessionAffinity.Locations }}{{ if eq (len $locationPaths) 1 }}{{ range $locationPath := $locationPaths }} path={{ $locationPath }}{{ end }}{{ end }}{{ end}} httponly;
```

Comment:
`if eq (len $locationPaths) 1 }}{{ range $locationPath := $locationPaths }}` why using a range if you expect a single element?